tests/js/server-test-caseholder.js

```diff
@@ -0,0 +1,713 @@
+/* Tests for the "express" and "router" module */
+"use strict";
+
+// We use just the request-handling bits of the kettle stack in our tests, but we include the whole thing to pick up the base grades
+require("../../node_modules/kettle");
+require("../../node_modules/kettle/lib/test/KettleTestUtils");
+
+var fluid        = fluid || require("infusion");
+var gpii         = fluid.registerNamespace("gpii");
+var jqUnit       = require("jqUnit");
+var fs           = require("fs");
+
+fluid.registerNamespace("gpii.express.couchuser.test.server.caseHolder");
+
+fluid.setLogging(true);
+
+gpii.express.couchuser.test.server.caseHolder.addRequiredSequences = function (sequenceStart, rawTests) {
```

Comment:
Glad you like it.  I think I copied and pasted it from code you gave me a while ago.